awscli/shorthand.py

```diff
@@ -348,27 +339,30 @@ def _expect(self, char, consume_whitespace=False):
         if consume_whitespace:
             self._consume_whitespace()
         if self._index >= len(self._input_value):
-            raise ShorthandParseSyntaxError(self._input_value, char,
-                                            'EOF', self._index)
+            raise ShorthandParseSyntaxError(
+                self._input_value, char, 'EOF', self._index
+            )
         actual = self._input_value[self._index]
         if actual != char:
-            raise ShorthandParseSyntaxError(self._input_value, char,
-                                            actual, self._index)
+            raise ShorthandParseSyntaxError(
+                self._input_value, char, actual, self._index
+            )
         self._index += 1
         if consume_whitespace:
             self._consume_whitespace()
 
     def _must_consume_regex(self, regex):
-        result = regex.match(self._input_value[self._index:])
+        result = regex.match(self._input_value[self._index :])
         if result is not None:
             return self._consume_matched_regex(result)
-        raise ShorthandParseSyntaxError(self._input_value, '<%s>' % regex.name,
-                                        '<none>', self._index)
+        raise ShorthandParseSyntaxError(
+            self._input_value, f'<regex.name>', '<none>', self._index
```

Comment:
this seems wrong. I would thing this is updated to `f'<{regex.name}>'` not `f'<regex.name>'`